src/main/java/tr/edu/ogu/ceng/service/CompanyService.java

```diff
@@ -2,21 +2,20 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+
 import tr.edu.ogu.ceng.dao.CompanyRepository;
 import tr.edu.ogu.ceng.model.Company;
-import tr.edu.ogu.ceng.dto.CompanyDto;
+import tr.edu.ogu.ceng.service.Exception.EntityNotFoundException;
 
 @Service
 public class CompanyService {
 
 	@Autowired
     private CompanyRepository companyRepository;
-
   
-    public CompanyDto getCompany(long id) {
-    	Company c = companyRepository.findById(id).orElse(null);
-    	if (c == null) return null;
-		return new CompanyDto(c.getId(), c.getName(), c.getAddress(), c.getPhoneNumber(), c.getFaxNumber(),c.getEmail(), c.getScope(), c.getDescription());
+    public Company getCompany(long id) throws EntityNotFoundException {
+    	Company company = companyRepository.findById(id).orElse(null);
+    	if (company == null) throw new EntityNotFoundException();
+		return company;
```

Comment:
indentation şurada o kadar önemli ki, kodun yanlış okunması için tuzak kurulmuş gibi sanki.

Reply:
one line if attığım için github öyle gösteriyor sanırım. idede normal gözüküyordu. yine de daha güzel göstermesi için düzenledim.